enhancements/installer/openstack-for-power-vc-ipi.md

```diff
@@ -0,0 +1,226 @@
+
+### Open Questions
+
+#### CI Plan
+Currently, Multi-Arch CI for Power connects to dedicated hardware provided by IBM and provisions clusters using IPI for
+libvirt, which isn't a supported customer configuration.
+
+We will create prow jobs that uses dedicated hardware that hosts a PowerVC cluster.
+
+### Test Plan
+
+Multi-Arch currently tests results by running upgrade and e2e tests with the OpenShift Prow CI environment. Our test
+plan would extend this to include these IPI scenarios. Additionally, new documentation will be needed for PowerVC,
+and the test plan will incorporate review and validation of this.
+
+### Graduation Criteria
+
+- Our initial target is to work towards a `Tech Preview`, then graduate towards future milestones. This is expected to
+take a total of two releases but could be more or less depending on how quickly we meet the requirements enumerated in
+the graduation criteria.
+- `Tech Preview` will graduate to `GA` in OpenShift
+- Our current target for `Tech Preview` is OpenShift 4.21, but this will be reevaluated as work progresses. Commitments
+are tracked in the Multi-Arch program sign-off documents for each release, which is created as part of Quarterly
+Planning.
+- [Deprecation policy][deprecation-policy]
+
+[deprecation-policy]: https://kubernetes.io/docs/reference/using-api/deprecation-policy/
+
+#### Dev Preview -> Tech Preview
+In 4.20 we intend to have an implementation behind a feature gate.
+
+#### Tech Preview Requirements
+
+- [ ] Full functionality / ability to utilize the enhancement end to end
+- [ ] End-user documentation
+- [ ] CI e2e jobs created to cover installation against this platform
+- [ ] Plan defined & support engaged to gather feedback from users
+
+#### Tech Preview -> GA
+- [ ] CI has been upgraded to cover testing of modifications to the installer (PR validation)
+- [ ] CI "soak" for to ensure a stable baseline
+- [ ] Final optimizations for install time
+- [ ] Expanded testing (upgrade, downgrade, scale, load/performance)
+- [ ] Documentation is finalized
+
+#### Removing a deprecated feature
+
+- Announce deprecation and support policy of the existing feature
+- Deprecate the feature
+
+### Upgrade / Downgrade Strategy
+
+All upgrade and downgrade expectations are the same for existing IPI deployments.
+
+### Version Skew Strategy
+
+This should be the same as other IPI deployments.
+
+### Operational Aspects of API Extensions
+N/a
+
+#### Failure Modes
+N/a
+
+#### Support Procedures
+N/a
+
+## Implementation History
+
+Major milestones in the life cycle of a proposal should be tracked in `Implementation History`.
+
+## Drawbacks
+
+## Alternatives (Not Implemented)
+
+We can continue to rely on our UPI playbooks but this is not ideal as they are not supported. We can also point users to
+our assisted installer support for PowerVC, but we believe users desire a more automated solution in this environment.
```

Comment:
We need to explore `platform: external`: 
Is it feasible to use `platform: external` and rely back on OpenStack as a provider? 
Comparing this alternative to the proposal (overhead, maintenance, tests and validation, future features) can also help.

Reply:
addressed with a few sentences in the next commit